config/app.json

```diff
@@ -6,6 +6,15 @@
     "inline",
     "column-2"
   ],
+  "animations": {
+    "none": "No Animation",
```

Comment:
It seems to be saving the value not the object name in the database

<img width="467" alt="Screenshot 2023-08-15 at 11 28 58" src="https://github.com/EddieHubCommunity/LinkFree/assets/624760/087f9910-f6b9-4707-a6f6-97002ea4d043">